src/components/Map.js

```diff
@@ -407,25 +402,55 @@ class MapComponent extends Component {
         latlng
       )
     );
-    
-    const wmsUrlsPromises = wmsUrls.map(url => getFromUrl(url)); 
-    Promise.all(wmsUrlsPromises).then(promiseResults => {
+    const urlsPromises = rasterUrls ? rasterUrls.map(url => getFromUrl(url)) : wmsUrls.map(url => getFromUrl(url));
```

Comment:
Looks good.
May not work if we want this feature to work for both rasters and wms layers within the same tile. Do we support this usecase?

Reply:
Changed! Now this feature works with both rasters and wms layers at the same time